docs/guides/plugins.md

```diff
@@ -107,9 +107,10 @@ MyPlugin = crescent.Plugin[MyBot, None]
 
 typing.reveal_type(MyPlugin().app)  # `MyBot`
 ```
-## Model
+## Sharing Objects Between Files
 
-Passing information between different files is hard. That's why crescent provides the `model` attribute. A `model` is any object you want that will be dependency-injected into your plugins.
+Sharing object between different files is hard, crescent provides the `model` attribute on plugins to solve this problem.
+A `model` is any object you want that will be dependency-injected into your plugins.
```

Comment:
"Sharing object between" is incorrect grammar. "Sharing objects" or "Sharing an object"

Same stuff as last comment, possibly "object" -> "state" & "files" -> "plugins".

"dependency-injected" -> "injected"?

Reply:
I think that makes more sense

Reply:
On second thought I think `dependency-injected` is the more accurate term here. If you really wanna change it just tell me cause I don't wanna bike shed too much.

Reply:
What about the first two suggestions? Idc much about "dependency-injected", though I think "injected" means the same thing in this context and is shorter.

Reply:
I made the other changes